astroquery/solarsystem/pds/tests/test_ringnode.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+from astropy.tests.helper import assert_quantity_allclose
+import numpy as np
+
+from ... import pds
+
+
+# Horizons has machinery here to mock request from a text file
+# is that necessary? why is that done?
+# wouldn't we want to know if the website we are querying changes something that makes code fail?
```

Comment:
Remote tests are not routinely run.  With a mock test, we can run many tests in a short period of time without concern for server load or even internet connection.

Reply:
ok, I'll do my best with offline tests